doc/quickstart/gmt_quickstart.rst

```diff
@@ -140,3 +141,14 @@ View results: (within :command:`gv` [#gv]_, type :kbd:`q` to quit)
 
 .. Rubric:: Footnotes
 .. [#gv] Ghostview  http://www.gnu.org/software/gv/
+
+
+What next?
+==========
+
+* Have a read of the local documentation stored in: `/usr/share/doc/gmt/html/ <../../gmt/html/index.html>`_
```

Comment:
aha - docs path moved here.. gotcha

Reply:
Yeah, I did rearrange some content. But please tell me if it doesn't make sense. I was guessing a bit.